core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -51,17 +51,27 @@ angular.module('oppia').factory('ThreadDataService', [
       suggestionThreads: []
     };
 
+    // TODO(brianrodri@): Use a helper-object to give this function O(1)
+    // complexity instead of O(N).
+    var getThreadWithId = function(threadId) {
```

Comment:
FWIW our usual nomenclature (at least in the backend) is getThreadById -- maybe rename for consistency.